[2025-03-10T08:27:49.691+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-10T08:27:50.552+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: validate_all_tables.orders manual__2025-03-10T08:27:04.851635+00:00 [queued]>
[2025-03-10T08:27:50.623+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: validate_all_tables.orders manual__2025-03-10T08:27:04.851635+00:00 [queued]>
[2025-03-10T08:27:50.636+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-10T08:27:50.786+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): orders> on 2025-03-10 08:27:04.851635+00:00
[2025-03-10T08:27:50.840+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4468) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-10T08:27:50.871+0000] {standard_task_runner.py:72} INFO - Started process 4472 to run task
[2025-03-10T08:27:50.871+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'validate_all_tables', 'orders', 'manual__2025-03-10T08:27:04.851635+00:00', '--job-id', '164', '--raw', '--subdir', 'DAGS_FOLDER/validate_all_tables_dag.py', '--cfg-path', '/tmp/tmpr3knzeuq']
[2025-03-10T08:27:50.892+0000] {standard_task_runner.py:105} INFO - Job 164: Subtask orders
[2025-03-10T08:27:51.222+0000] {task_command.py:467} INFO - Running <TaskInstance: validate_all_tables.orders manual__2025-03-10T08:27:04.851635+00:00 [running]> on host efacfdd0be48
[2025-03-10T08:27:51.825+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='validate_all_tables' AIRFLOW_CTX_TASK_ID='orders' AIRFLOW_CTX_EXECUTION_DATE='2025-03-10T08:27:04.851635+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-10T08:27:04.851635+00:00'
[2025-03-10T08:27:51.842+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-10T08:27:51.856+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-10T08:27:51.858+0000] {logging_mixin.py:190} INFO - Current task name:orders state:running start_date:2025-03-10 08:27:50.555853+00:00
[2025-03-10T08:27:51.859+0000] {logging_mixin.py:190} INFO - Dag name:validate_all_tables and current dag run status:running
[2025-03-10T08:27:51.872+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-10T08:27:52.260+0000] {logging_mixin.py:190} INFO - Orders validation and cleaning successful
[2025-03-10T08:27:52.276+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-10T08:27:52.328+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-10T08:27:52.338+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=validate_all_tables, task_id=orders, run_id=manual__2025-03-10T08:27:04.851635+00:00, execution_date=20250310T082704, start_date=20250310T082750, end_date=20250310T082752
[2025-03-10T08:27:52.476+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-10T08:27:52.486+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-10T08:27:52.487+0000] {logging_mixin.py:190} INFO - Dag name:validate_all_tables queued_at:2025-03-10 08:27:04.920638+00:00
[2025-03-10T08:27:52.488+0000] {logging_mixin.py:190} INFO - Task hostname:efacfdd0be48 operator:PythonOperator
[2025-03-10T08:27:52.566+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-10T08:27:52.951+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-10T08:27:52.987+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
